asapdiscovery-alchemy/asapdiscovery/alchemy/predict.py

```diff
@@ -989,3 +989,76 @@ def download_cdd_data(protocol_name: str) -> pd.DataFrame:
     )
 
     return formatted_data
+
+
+def clean_result_network(network, console=None):
+    """
+    Cleans an incoming result network JSON file from some issues that might occur. Current procedures:
```

Comment:
This feels like a bit of a workaround. We should try and identify issues upstream causing this and eliminate them if possible.

Reply:
100%. `-c` is False by default so the user is prompted every time it's required (and every one of those times things should actually get fixed upstream)